integration-test/pom.xml

```diff
@@ -167,31 +167,25 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_${scala.binary.version}</artifactId>
+      <scope>test</scope>
```

Comment:
Can you point out which code requires spark-core, AFAIK the only requirement is JobAPI tests.